utests/test_splines.c

```diff
@@ -191,8 +191,8 @@ static bool test_bspline_zero(void)
 		double coord[7] = { 0., 0., 0., 0., 0., 0., 0. };
 		coord[i] = 1.;
 
-		double k2[9];
-		double c2[6];
+		double k2[9] = {0., 0., 0., 0., 0., 0., 0., 0., 0.};
```

Comment:
I have to look at the underlying issue. Maybe something is broken in the spline code...

Reply:
That error was really weird, it only happened when using `clang` to compile BART.

Reply:
Similar to the other one, but that turned out to be a bug in BART.